Open-Rival/include/UnitRenderer.h

```diff
@@ -31,6 +31,8 @@ namespace Rival {
                 std::map<UnitType, Spritesheet>& unitSprites,
                 Texture& paletteTexture);
 
+        UnitRenderer& operator=(const UnitRenderer&) = delete;
```

Comment:
Do we also need to implement / delete the destructor / copy constructor here (rule of 5) ?